recipe/recipe.yaml

```
@@ -14,15 +14,15 @@ source:
 build:
   noarch: python
   script: python -m pip install . -vv
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=3.9
+    - python ${{ python_min }}
```

The patch version can float. So `.*` would do what we need

```suggestion
    - python ${{ python_min }}.*
```